lib/ae_mdw/contract.ex

```diff
@@ -473,6 +473,14 @@ defmodule AeMdw.Contract do
     }
   end
 
+  defp element_value({type, [head | _tail] = keyword_list})
+       when is_atom(type) and is_tuple(head) and tuple_size(head) == 2 do
```

Comment:
Is is guaranteed that if the first element is a tuple of 2 values then the rest will also be?

Reply:
For `type = :tuple` I would expect that the SDK encoding is uniform. The code is more inclusive because if `type != :tuple` and `keyword_list` is any list of pairs it's able to render it. Any other encoding would require it's own investigation.

Reply:
Makes sense. Thanks